src/sentry/models/organization.py

```diff
@@ -120,6 +122,53 @@ def get_for_user(self, user, scope=None, only_visible=True):
             return [r.organization for r in results if scope in r.get_scopes()]
         return [r.organization for r in results]
 
+    def get_member_top_dog_organizations(self, user_id: int, queryset: QuerySet = None) -> QuerySet:
+        from sentry.models import OrganizationMember, OrganizationMemberTeam, Team
+
+        # get orgs and orgmemberIDs for the user
+        if queryset:  # queryset is a QuerySet of valid orgs
+            query = list(
```

Comment:
Once you listify it, its no longer a query. ( believe this triggers the db query)

rename?